app/controllers/application_controller.rb

```diff
@@ -2,4 +2,9 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  before_filter :set_app_info_headers
+
+  def set_app_info_headers
+  	headers['X-GitSHA'] = AppInfo::GIT_REVISION
```

Comment:
sure; I was also thinking about adding a header for the github repo itself (since it is open source) - thoughts?